index.html

```diff
@@ -4,11 +4,32 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link rel="stylesheet" href="body.css">
-
-    <h1>"Hello Microverse!"</h1>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <title>Frehiywot Nega Portifolio</title>
         
   </head>
   <body>
+    <section class="navigation">
+      <header class="mobile_header">
+        <div class="image_geometry_1"> </div>
+        <nav class="menu-button-selector">
+          <img src="image/ic_menu@2x.jpg" alt="menu" class="menu-button" />
```

Comment:
Great job @frnega 🤝

- [ ] [Optional] You may surround this icon with an `<a>` tag. This is because this icon is actually a clickable link.